WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -288,7 +279,6 @@ public static void deleteDatabase(Context ctx) {
     public void dangerouslyDeleteAllContent() {
         db.delete(MEDIA_TABLE, null, null);
         db.delete(CATEGORIES_TABLE, null, null);
-        db.delete(CommentTable.COMMENTS_TABLE, null, null);
```

Comment:
We'll have to re implement `dangerouslyDeleteAllContent` someway - we actually have a ticket for this: https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/76